elife/python3.sls

```diff
@@ -14,9 +19,17 @@ python-3:
             
     pkg.installed:
         - pkgs:
+            # nothing uses python 3.4 or 3.6
+            # django has/had issues with 3.6 when last tested so we settled on 3.5
+            # 3.6 is expected to be available in 18.04 but not as default
             - python3.4
             - python3.4-dev
             - python3.5
             - python3.5-dev
             - python3.6
             - python3.6-dev
+
+            # ubuntu ... ffs.
+            # https://bugs.launchpad.net/ubuntu/+source/python3.4/+bug/1290847
+            - python3.5-venv
```

Comment:
this issue just caught me out while building lax on 14.04 with the this branch. not sure if you've seen the error yet, but it breaks with a message like this: 

`Error: Command '['/srv/lax/venv/bin/python3.5', '-Im', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1`